SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -899,6 +922,163 @@ FString FSpatialGDKEditorToolbarModule::GetOptionalExposedRuntimeIP() const
 	}
 }
 
+FReply FSpatialGDKEditorToolbarModule::OnLaunchDeployment()
```

Comment:
This function is moved from `SpatialGDKSimulatedPlayerDeployment.cpp`, which isn't strictly required for this PR but it makes the changes smaller in the next one which will add a button to quick deploy without opening the cloud deployment window if it has already been configured.